app/Notifications/RequestNCPOForApprovalNotification.php

```
@@ -0,0 +1,72 @@
+use App\Models\RequestNCPO;
+use Illuminate\Bus\Queueable;
+use Illuminate\Notifications\Messages\MailMessage;
+use Notification;
+
+class RequestNCPOForApprovalNotification extends Notification
+{
+    use Queueable;
+
+    private $token;
+    private $model;
+    public $id;
```

PHPMD (2.15.0)

Avoid variables with short names like $id. Configured minimum length is 3. (undefined)

(ShortVariable)